src/thumbnailer.h

```diff
@@ -84,6 +86,11 @@ class Thumbnailer {
 
   Status SetLoopCount(WebPData* const webp_data);
 
+  // Takes two thumbnails as WebPData. Prints various statistics regarding
```

Comment:
`CompareThumbnail()` should output the stats rather than print them. It is the job of main.cc (or another binary) to print or aggregate them.